example/ConnectionInfoSubscription.tsx

```diff
@@ -45,7 +45,7 @@ export default class ConnectionInfoSubscription extends React.Component<
     return (
       <View>
         <Text style={{color: 'black'}}>
-          {JSON.stringify(this.state.connectionInfoHistory)}
+          {JSON.stringify(this.state.connectionInfoHistory, null, 4)}
```

Comment:
This may seem like a silly preference but the reason they are not pretty-printed is so that during testing on emulators etc it all fits in one window easy to see, no scrolling

```suggestion
          {JSON.stringify(this.state.connectionInfoHistory)}
```